app/models/payments/ChargeType.scala

```diff
@@ -305,6 +305,26 @@ case object VatReturn2ndLPP extends ChargeType {
   override val value: String = "VAT Return 2nd LPP"
 }
 
+case object VatErrorCorrectionLPI extends ChargeType {
+  override val value: String = "VAT Error Correction LPI"
+}
+
+case object VatErrorCorrection1stLPP extends ChargeType {
+  override val value: String = "VAT Error Correction 1st LPP"
+}
+
+case object VatErrorCorrection2ndLPP extends ChargeType {
+  override val value: String = "VAT Error Correction 2nd LPP"
+}
+
+case object VatErrorCorrection1stLPPLPI extends ChargeType {
+  override val value: String = "VAT Error Correction 1st LPP LPI"
```

Comment:
The charge type value for this and the LPI below should be `Correct` instead of `Correction`